.jscsrc

```diff
@@ -50,6 +50,8 @@
 		"examples/typescript-*/js/**/*.js",
 		"examples/vanilladart/**/*.js",
 		"examples/vanilla-es6/dist/bundle.js",
+		"examples/react-redux/js/reducers/*.js",
```

Comment:
I tried it, but it didn't work. The only way I found to make it work is with the `esnext` flag.